app/views/posts/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_for ([current_user, current_user.posts.build]) do |form| %>
```

Comment:
Nice use of a Rails convention for doing complicated things with small amounts of code.